.travis.yml

```
@@ -6,6 +6,8 @@ branches:
   only:
     - develop
     - master
+    - metrolist_dev
+    - metrolist_ci
```

Should I remove these: 

```suggestion
    - metrolist_dev
    - metrolist_ci
```

---

Removed